packages/main/src/plugin/docker/docker-compatibility.spec.ts

```diff
@@ -44,8 +96,204 @@ test('should register a configuration', async () => {
   expect(configurationNode?.title).toBe('Experimental (Docker Compatibility)');
   expect(configurationNode?.properties).toBeDefined();
   expect(Object.keys(configurationNode?.properties ?? {}).length).toBe(1);
-  expect(configurationNode?.properties?.[DockerCompatibility.ENABLED_FULL_KEY]).toBeDefined();
-  expect(configurationNode?.properties?.[DockerCompatibility.ENABLED_FULL_KEY]?.type).toBe('boolean');
-  expect(configurationNode?.properties?.[DockerCompatibility.ENABLED_FULL_KEY]?.default).toBeFalsy();
-  expect(configurationNode?.properties?.[DockerCompatibility.ENABLED_FULL_KEY]?.hidden).toBeTruthy();
+  expect(configurationNode?.properties?.[TestDockerCompatibility.ENABLED_FULL_KEY]).toBeDefined();
+  expect(configurationNode?.properties?.[TestDockerCompatibility.ENABLED_FULL_KEY]?.type).toBe('boolean');
+  expect(configurationNode?.properties?.[TestDockerCompatibility.ENABLED_FULL_KEY]?.default).toBeFalsy();
+  expect(configurationNode?.properties?.[TestDockerCompatibility.ENABLED_FULL_KEY]?.hidden).toBeTruthy();
+});
+
+describe('getTypeFromServerInfo', async () => {
+  test('Docker Desktop', async () => {
+    const dockerCompatibility = new TestDockerCompatibility(configurationRegistry, providerRegistry);
+    const serverInfo = {
+      OperatingSystem: 'Docker Desktop',
+    };
+
+    const podmanInfo = {};
+
+    expect(dockerCompatibility.getTypeFromServerInfo(serverInfo, podmanInfo)).toBe('docker');
```

Comment:
```suggestion
    expect(dockerCompatibility.getTypeFromServerInfo(serverInfo, {})).toBe('docker');
```

No eslint complain ?

Reply:
why should it complain, it's an empty object.

eslint complain about empty interface or empty function

Reply:
removed the object and made the podmanInfo parameter optional